nxdrive/client/remote_client.py

```diff
@@ -97,6 +97,10 @@ def __init__(
                 "X-User-Id": user_id,
                 "X-Device-Id": device_id,
                 "Cache-Control": "no-cache",
+                "NX-metric-installation.type": "system-wide"
+                if Options.system_wide
+                else "user-only",
```

Comment:
```suggestion
                "NX-metric-installation.type": (
                    "system-wide" if Options.system_wide else "user-only"
                ),
```
Or move it to a variable, it is too ugly as-is.